analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -212,7 +212,7 @@ class GoMod(
 
     private fun ModuleInfo.toId(): Identifier =
         Identifier(
-            type = managerName.takeIf { version.isBlank() } ?: "Go",
+            type = managerName.takeIf { main } ?: "Go",
```

Comment:
Commit message nits:
- "determining *a* value"
- "without *a* version" (twice)
- Probably a comma before "this change is of non-functional nature"

Reply:
Not strictly related, but how do you think about renaming the `main` and `indirect` boolean properties to `isMain` and `isIndirect` in another commit for clarity?

Reply:
The names are exactly the same as in `golang` upstream data structures. I find it a tad simpler if these are consistent, which is why I'd prefer to keep these names.

Reply:
> The names are exactly the same as in `golang` upstream data structures.

That's not true, as they already differ in capitalization.

Reply:
I think you got my point.